server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -146,6 +146,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion ERROR_TRACE_IN_TRANSPORT_HEADER = def(8_811_00_0);
     public static final TransportVersion FAILURE_STORE_ENABLED_BY_CLUSTER_SETTING = def(8_812_00_0);
     public static final TransportVersion SIMULATE_IGNORED_FIELDS = def(8_813_00_0);
+    public static final TransportVersion ESQL_TELEMETRY_STATS = def(8_814_00_0);
```

Comment:
This should probably be `ESQL_CCS_TELEMETRY_STATS` since there is other non-ccs telemetry.